src/app/inventory/store/d2-item-factory.ts

```diff
@@ -528,6 +529,18 @@ export function makeItem(
     }
   }
 
+  // a weapon can have an inherent breaker type, or gain one from socketed mods
+  // (or armor mods can sort of add them but let's not go there quite yet)
+  // this is presented as an else-type dichotomy here, but who knows what the future holds
+  if (itemDef.breakerTypeHash)
+    createdItem.breakerType = defs.BreakerType.get(itemDef.breakerTypeHash);
+  else if (createdItem.bucket.inWeapons && createdItem.sockets) {
```

Comment:
Braces around expression (prettier should have gotten this?)

Reply:
do we have a specific prettier rule for it? i don't see it at a glance and my vs isn't picking it up.
it's autoformatting but not applying that specific rule.